sched/job_def_serializer_test.go

```diff
@@ -1,3 +1,5 @@
+// +build property
```

Comment:
This file has property & unit tests in it.  adding this tag will make interpret all of them as property based tests. Can you split the property tests out into a separate file?